packages/medusa-payment-adyen/src/services/adyen.js

```diff
@@ -1,12 +1,12 @@
 import axios from "axios"
 import _ from "lodash"
 import { hmacValidator } from "@adyen/api-library"
-import { BaseService } from "medusa-interfaces"
+import { TransactionBaseService, PaymentSessionStatus } from "@medusajs/medusa"
 import { Client, Config, CheckoutAPI } from "@adyen/api-library"
 
-class AdyenService extends BaseService {
+class AdyenService extends TransactionBaseService {
```

Comment:
issue: Should extend the abstract payment service
```suggestion
class AdyenService extends AbstractPaymentService {
```